run/jenkins_test.py

```diff
@@ -14,14 +14,13 @@
 
```

Comment:
Can we move this to a test folder?

Reply:
Pardon my Python ignorance: Would that require us to use relative imports that include `..` or something? If someone can help me sort out how the test code refers to the code it's testing I'm happy to make the change. @jeffcarp is there a reason we have been using _test suffix instead of a test dir?

Reply:
Whoops, I misread when I looked at the code before. I didn't see we already had ```_test``` suffixed files.
In that case, I think we should merge this PR with the file organization as is here and change browsers_test and run_test in another PR afterwards. I will open an issue.